extensions/librdkafka/PublishKafka.cpp

```diff
@@ -184,12 +184,12 @@ class ReadCallback {
     });
   }
 
-  static rd_kafka_headers_unique_ptr make_headers(const core::FlowFile& flow_file, utils::Regex& attribute_name_regex) {
+  static rd_kafka_headers_unique_ptr make_headers(const core::FlowFile& flow_file, std::optional<utils::Regex>& attribute_name_regex) {
```

Comment:
could this be `const optional<Regex>&`?  (also at line 241)

Reply:
Yes it can, updated in 07973bdddd5299402d6c0c7a2e71ff7e0fc39e67